addons/Popochiu/Engine/Objects/Room/PopochiuRoom.gd

```diff
@@ -253,6 +243,22 @@ func set_active_walkable_area(walkable_area_name: String) -> void:
 		[script_name, walkable_area_name])
 
 
+func get_characters() -> Array:
+	var characters := []
+	
+	for c in $Characters.get_children():
```

Comment:
Godot 4 will have filter methods for that it seems.

Reply:
That will be great!

Reply:
Sure, was more of a note to self than a real comment :)